brave/brave-core
# feat(wallet) Add Coinbase Pay on-ramp

Resolves https://github.com/brave/brave-browser/issues/32083



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
- Go to brave://wallet
- Click on Buy
- On the buy page, select a token e.g Ethereum
- Click on Continue
- On the onramp page, Coinbase Pay should listed as an option
- Select coinbase pay
- A new tab opens and loads coinbase pay widget
